test/com/google/javascript/jscomp/CollapsePropertiesTest.java

```diff
@@ -35,26 +37,45 @@
       + "/** @constructor */ function String() {};\n"
       + "var arguments";
 
+  private boolean processCommonJsModules = false;
+  private boolean collapseModuleExportProperties = false;
+
   public CollapsePropertiesTest() {
     super(EXTERNS);
   }
 
   @Override
   protected CompilerPass getProcessor(final Compiler compiler) {
-    return new CollapseProperties(compiler);
+    PropertyType propertyType = PropertyType.ANY;
+    if (this.collapseModuleExportProperties) {
+      propertyType = PropertyType.MODULE_EXPORT;
+    }
+    return new CollapseProperties(compiler, propertyType);
   }
 
   @Override
   protected void setUp() throws Exception {
     super.setUp();
     enableNormalize();
+    if (this.collapseModuleExportProperties) {
+      this.setAcceptedLanguage(LanguageMode.ECMASCRIPT_2015);
+      this.setLanguageOut(LanguageMode.ECMASCRIPT5);
+    }
```

Comment:
I think this part is redundant with the settings in setupModuleExportsOnly below, isn't it?